tests/integration_tests/style/test_rust.py

```diff
@@ -27,34 +30,28 @@ def test_ensure_mod_tests():
 
     @type: style
     """
-    # List all source files containing rust #[test] attribute,
-    # (excluding generated files and integration test directories).
-    # Take the list and check each file contains 'mod tests {', output file
-    # name if it doesn't.
-    cmd = (
-        "/bin/bash "
-        "-c "
-        '"grep '
-        "--files-without-match "
-        "'mod tests {' "
-        "\\$(grep "
-        "--files-with-matches "
-        "--recursive "
-        "--exclude-dir=src/*_gen/* "
-        "'\\#\\[test\\]' ../src/*/src)\" "
-        '| grep -v "../src/io_uring/src/bindings.rs"'
+    excluding = [
+        "_gen/",
+        "/tests/",
+        "/test_utils",
+        "build/",
+        "src/io_uring/src/bindings.rs",
+    ]
+
+    def cmd(string):
+        cmd = shlex.split(string)
```

Comment:
don't need to `shlex` if you use `shell=True` I think.

Reply:
Fixed.